### PR TITLE
Static site

### PR DESCRIPTION
**Pull request by mrerrormessage**
- `/tortoise` can be used with a static backing server
- Automatic deploys to THE CLOUD
- When NetLogo code is posted to the server (as opposed to an nlogo file) it uses the key `"code"` instead of `"model"`, which allows requests to look the same as responses.

EDITED (per @TheBizzle 's suggestion):
:cloud: :cloud: :cloud: :cloud: :cloud: :cloud: :cloud: :cloud: :cloud: :cloud: :cloud: :cloud:

### CODE REVIEW

**Comment by TheBizzle** [comment]
I appreciate the amount of THE CLOUD in this pull request, but I think it's important that we get a bit more of THE CLOUD into this PR before I can merge it.  Sorry to rain on your parade.

**Comment by qiemem** [comment]
> When NetLogo code is posted to the server (as opposed to an nlogo file) it uses the key "code" instead of "model", which allows requests to look the same as responses.

Both request and response should use `model`. I chose `model` because it worked for all request types (request method distinguishes the type).

**Comment by qiemem** [comment]
Oh, I see where you're coming from on the `code` thing. `code` is include in the `url` and `nlogo` responses because its the server that parses out the code... hmm... Well, whatever, I guess it's fine.

File: app/assets/javascripts/TortoiseJS/control/session-lite.coffee [hunk 1]
```diff
@@ -78,14 +78,19 @@ class window.SessionLite
   exportnlogo: ->
```
Comment (qiemem) [inline app/assets/javascripts/TortoiseJS/control/session-lite.coffee]: `exportNlogo`

**Comment by qiemem** [comment]
lgtm!

**Pull request merged**